### DISCUSSION TITLE
$inPreview function like $inEditor

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1708] [General]
Is there a function to tell if you are in the editor preview to stop scripts executing? I want this in a main layout file and not a block.

Like the block editor has $inEditor

**Comment by haringsrob** [comment]
Hey @michaeljcoyne,

You could use "previewData" maybe? https://twill.io/docs/crud-modules/revisions-and-previewing.html#revisions-and-previewing
